documentation/modules/proc-sending-messages.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-using-amq-streams-{context}']
+
+= Sending and receiving messages from a topic
+
+Test your {ProductName} installation by sending and receiving messages outside the cluster from `my-topic`.
+
+Use a terminal to run a Kafka producer and consumer on a local machine.
+
+.Prerequisites
+
+* {ProductName} is installed on the Kubernetes cluster
+* Zookeeper and Kafka are running
+
+.Procedure
+
+. Download the latest Kafka binaries and install Kafka on your local machine.
++
+{ApacheKafkaDownload}
+
+. Find the port of the bootstrap service:
++
+[source, shell, subs=+quotes, options="nowrap"]
+----
+kubectl get service my-cluster-kafka-external-bootstrap -n my-kafka-project -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}'
+----
+
+. Find the IP address of the Minikube node:
++
+[source, shell, subs=+quotes, options="nowrap"]
+----
+kubectl get nodes --output=jsonpath='{range .items[*]}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}'
+----
+
+. Open a terminal and start the Kafka console producer with the topic `my-topic` and the authentication properties for TLS:
++
+[source,shell,subs=+quotes]
+----
+bin/kafka-console-producer.sh --broker-list _<node-address>_:_<node-port>_ --topic my-topic
```

Comment:
We mention the "authentication properties for TLS" but there's nothing here which configures the producer to use TLS.

Reply:
Ah, residual from Jakub's switch to minikube/nodeport. Removed.